### PR TITLE
Fix table toolbar select height

### PR DESCRIPTION
**Pull request by rcfish**
The table toolbar select when displayed expands to 82px in height (vs the 64px for the standard toolbar). This has the effect that the overall table height is changed and users see the table data shift up or down. Now the table toolbar select remains at 64px and there is no data shift up or down.

### CODE REVIEW

File: src/components/TableToolbarSelect.js [hunk 1]
```diff
@@ -12,26 +12,18 @@ const defaultToolbarSelectStyles = theme => ({
...
     position: 'relative',
     zIndex: 120,
     justifyContent: 'space-between',
+    alignItems: 'center',
     paddingTop: theme.spacing.unit,
     paddingBottom: theme.spacing.unit,
   },
   title: {
     paddingLeft: '26px',
-    top: '50%',
```
Comment (gabrielliwerant) [inline src/components/TableToolbarSelect.js]: I think we might need these styles under `title`. If you take a look at this with the `customize-toolbarselect` example, where there are multiple icons in a custom toolbar, you'll see that they don't line up without them.

Comment (rcfish) [reply src/components/TableToolbarSelect.js]: Thanks. Will check it out and update.

Comment (rcfish) [reply src/components/TableToolbarSelect.js]: @gabrielliwerant I don't think we have an issue here. The `customize-toolbarselect` example has the same `iconButton` styling as the default component which is the styling I removed. Removing it from the example and the icons all line up with the title.

Comment (rcfish) [reply src/components/TableToolbarSelect.js]: image

Comment (gabrielliwerant) [reply src/components/TableToolbarSelect.js]: Lets remove that styling from the example then, so that it is not broken.

Comment (rcfish) [reply src/components/TableToolbarSelect.js]: Example updated

Comment (gabrielliwerant) [reply src/components/TableToolbarSelect.js]: Thank you!

**Comment by rcfish** [comment]
I'm happy to revert back the webpack file, wasn't sure what the protocol was for this.

**Pull request merged**